README.md

```diff
@@ -1,3 +1,134 @@
+
+```
+---
+spec_version: '0.1.0'
+workloads:
+  - teststrategy: functional
+    script: /locust/basic.py
+    conf: /locust/locust.conf
+  - teststrategy: performance_light
+    script: /locust/basic.py
+    conf: /locust/locust.conf
+  - teststrategy: functional-light
+    script: /locust/basic.py
+```
+
+The contents of `locust.conf.yaml` should be used as described in the following:
+- If a "script" is given, it has to be used in the execution of the locust test.
+- If "script" is not given, we can assume that the script is part of the "config" file that is referenced.
+- If a "conf" is given, it should be used in the execution of the locust test.
+- If "conf" is not given, the "script" will be executed with default setting.
+- If both "script" and "conf" are missing, the integration skips the tests and indicate this in the result that is sent back to Keptn.
+
+Examples for both the `locust.conf.yaml` and the [locust config file](https://docs.locust.io/en/stable/configuration.html#configuration-file) can be found in the [test-data/](test-data) directory.
+
+
+## Uninstall -  Delete from your Kubernetes cluster
+
+To delete the locust-service, delete using the [`deploy/service.yaml`](deploy/service.yaml) file:
+
+```console
+kubectl delete -f deploy/service.yaml
+```
+
+## Development
+
+Development can be conducted using any Golang compatible IDE/editor (e.g., Jetbrains GoLand, VSCode with Go plugins).
+
+It is recommended to make use of branches as follows:
+
+* `master` contains the latest potentially unstable version
+* `release-*` contains a stable version of the service (e.g., `release-0.1.0` contains version 0.1.0)
+* create a new branch for any changes that you are working on, e.g., `feature/my-cool-stuff` or `bug/overflow`
+* once ready, create a pull request from that branch back to the `master` branch
+
+When writing code, it is recommended to follow the coding style suggested by the [Golang community](https://github.com/golang/go/wiki/CodeReviewComments).
+
+### Where to start
+
+If you don't care about the details, your first entrypoint is [eventhandlers.go](eventhandlers.go). Within this file you can add implementation for pre-defined Keptn Cloud events.
+ 
+To better understand Keptn CloudEvents, please look at the [Keptn Spec](https://github.com/keptn/spec).
+ 
+If you want to get more insights, please look into [main.go](main.go), [deploy/service.yaml](deploy/service.yaml), consult the [Keptn docs](https://keptn.sh/docs/) as well as existing [Keptn Core](https://github.com/keptn/keptn) and [Keptn Contrib](https://github.com/keptn-contrib/) services.
+
+### Common tasks
+
+* Run tests: `go test -race -v ./...`
+* Deploy the service using `kubectl`: `kubectl apply -f deploy/`
+* Delete/undeploy the service using `kubectl`: `kubectl delete -f deploy/`
+* Watch the deployment using `kubectl`: `kubectl -n keptn get deployment locust-service -o wide`
+* Get logs using `kubectl`: `kubectl -n keptn logs deployment/locust-service -f`
```

Comment:
does this work or is it 
`kubectl -n keptn logs deployment/locust-service locust-service -f` (including the container name)? please verify